kaspresso/src/main/kotlin/com/kaspersky/kaspresso/docloc/rule/ThemeRule.kt

```diff
@@ -0,0 +1,32 @@
+package com.kaspersky.kaspresso.docloc.rule
+
+import androidx.appcompat.app.AppCompatDelegate
+import com.kaspersky.kaspresso.logger.UiTestLogger
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class ThemeRule internal constructor(
```

Comment:
I would narrow down the name to tell user that this rule is used to toggle dark mode, not the whole theme

Reply:
i've changed